strawberry/types/types.py

```diff
@@ -29,9 +29,9 @@ class TypeDefinition:
     _type_params: Dict[str, Type] = dataclasses.field(default_factory=dict, init=False)
 
     def get_field(self, name: str) -> Optional["StrawberryField"]:
-        return next(
-            (field for field in self.fields if field.graphql_name == name), None
-        )
+        # TODO: check if python name is ok for field name or if we should check
+        # the graphql name
+        return next((field for field in self.fields if field.python_name == name), None)
```

Comment:
Is this a TODO for a different PR?

Reply:
mmh, maybe not :) I think we use this in the printer, so it's better if I check this before merging